hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapSetMessageTask.java

```diff
@@ -34,7 +35,9 @@ public MapSetMessageTask(ClientMessage clientMessage, Node node, Connection conn
 
     @Override
     protected Operation prepareOperation() {
-        SetOperation op = new SetOperation(parameters.name, parameters.key, parameters.value, parameters.ttl);
+        MapOperationProvider operationProvider = getMapOperationProvider(parameters.name);
```

Comment:
Perhaps an idea to rename parameters to params? Not part of this PR, but having wide lines is imho just as bad as having long methods.